locales/nl.json

```diff
@@ -1,315 +1,315 @@
+    "Irish": "Iers",
+    "Italian": "Italiaans",
+    "Japanese": "Japans",
+    "Javanese": "Javaans",
+    "Kannada": "Kannada",
+    "Kazakh": "Kazachs",
+    "Khmer": "Khmer",
+    "Korean": "Koreaans",
+    "Kurdish": "Koerdisch",
+    "Kyrgyz": "Kirgizisch",
+    "Lao": "Laotiaans",
+    "Latin": "Latijns",
+    "Latvian": "Lets",
+    "Lithuanian": "Litouws",
+    "Luxembourgish": "Luxemburgs",
+    "Macedonian": "Macedonisch",
+    "Malagasy": "Malagassisch",
+    "Malay": "Maleisisch",
+    "Malayalam": "Malayalam",
+    "Maltese": "Maltees",
+    "Maori": "Maorisch",
+    "Marathi": "Marathi",
+    "Mongolian": "Mongools",
+    "Nepali": "Nepalees",
+    "Norwegian Bokmål": "Noors (Bokmål)",
+    "Nyanja": "Nyanja",
+    "Pashto": "Pashto",
+    "Persian": "Perzisch",
+    "Polish": "Pools",
+    "Portuguese": "Portugees",
+    "Punjabi": "Punjabi",
+    "Romanian": "Roemeens",
+    "Russian": "Russisch",
+    "Samoan": "Samoaans",
+    "Scottish Gaelic": "Schots-Gaelisch",
+    "Serbian": "Servisch",
+    "Shona": "Shona",
+    "Sindhi": "Sindhi",
+    "Sinhala": "Sinhala",
+    "Slovak": "Slowaaks",
+    "Slovenian": "Sloveens",
+    "Somali": "Somalisch",
+    "Southern Sotho": "Zuid-Sotho",
+    "Spanish": "Spaans",
+    "Spanish (Latin America)": "Spaans (Latijns-Amerika)",
+    "Sundanese": "Soedanees",
+    "Swahili": "Swahili",
+    "Swedish": "Zweeds",
+    "Tajik": "Tajik",
+    "Tamil": "Tamil",
+    "Telugu": "Telugu",
+    "Thai": "Thaïs",
+    "Turkish": "Turks",
+    "Ukrainian": "Oekraïens",
+    "Urdu": "Urdu",
+    "Uzbek": "Oezbeeks",
+    "Vietnamese": "Vietnamees",
+    "Welsh": "Welsh",
+    "Western Frisian": "Fries",
+    "Xhosa": "Xhosa",
+    "Yiddish": "Joods",
+    "Yoruba": "Yoruba",
+    "Zulu": "Zulu",
     "`x` years": "`x` jaar",
     "`x` months": "`x` maanden",
     "`x` weeks": "`x` weken",
     "`x` days": "`x` dagen",
     "`x` hours": "`x` uur",
     "`x` minutes": "`x` minuten",
     "`x` seconds": "`x` seconden",
-    "Fallback comments: ": "",
-    "Popular": "",
-    "Top": "",
-    "About": "",
-    "Rating: ": "",
-    "Language: ": "",
-    "View as playlist": "",
-    "Default": "",
-    "Music": "",
-    "Gaming": "",
-    "News": "",
-    "Movies": "",
-    "Download": "",
-    "Download as: ": "",
-    "%A %B %-d, %Y": "",
-    "(edited)": "",
-    "YouTube comment permalink": "",
-    "`x` marked it with a ❤": "",
-    "Audio mode": "",
-    "Video mode": "",
-    "Videos": "",
-    "Playlists": "",
-    "Current version: ": ""
-}
\ No newline at end of file
+    "Fallback comments: ": "Terugvallen op",
+    "Popular": "Populair",
+    "Top": "Top",
+    "About": "Over",
+    "Rating: ": "Waardering",
+    "Language: ": "Taal",
+    "View as playlist": "Tonen als afspeellijst",
+    "Default": "Standaard",
+    "Music": "Muziek",
+    "Gaming": "Gaming",
+    "News": "Nieuws",
+    "Movies": "Films",
+    "Download": "Downloaden",
+    "Download as: ": "Downloaden als",
```

Comment:
Should this be `Downloaden als: `, or is there a reason we leave out the colon here?

Reply:
Oops, you're right: I accidentally missed the colon there.